.circleci/config.yml

```diff
@@ -26,6 +28,8 @@ jobs:
     executor:
       name: solidusio_extensions/mysql
       ruby_version: "3.1"
+    environment:
+      FRONTEND: "/home/circleci/project/spec/solidus_frontend_app_template.rb"
```

Comment:
Can you please better help me understand why this is needed?

Reply:
Yes, that's because up to Solidus v3.4 solidus_frontend was available as an option, and by just adding the gem to the gemfile we were able to trick the solidus installer into using it, having removed it in v4 we need a custom app template to install it. Rather than having different behaviors for v3 and v4 we just use the same recipe (which is fairly simple anyway).

Reply:
Got it. Sounds like something we will need to use for all extensions that were testing against the old frontend at some point.